src/components/FilterArea/FilterArea.jsx

```diff
@@ -0,0 +1,17 @@
+import "./FilterArea.css";
+import { Chips } from "../Chips/Chips";
+import { useDataValues } from "../../contextAndReducers/DataProvider";
+function FilterArea() {
+  const { dispatch } = useDataValues();
+  return (
+    <div className="FilterArea">
+      <Chips name={"Home"} />
+      <Chips name={"American Classics"} />
+      <Chips name={"Car Reviews"} />
+      <Chips name={"Custom"} />
+      <Chips name={"Repairs"} />
+      <Chips name={"Bikes"} />
```

Comment:
Here, you can map over the categories since you get them from the db. Hard coding the categories is not right.

Reply:
Yeah I will change it as suggested by you